xmrstak/backend/nvidia/nvcc_code/cuda_extra.cu

```diff
@@ -490,7 +498,7 @@ extern "C" int cuda_get_deviceinfo(nvid_ctx* ctx)
 				minSupportedArch = arch[i];
 		if(minSupportedArch < 30 || gpuArch < minSupportedArch)
 		{
-			printf("WARNING: NVIDIA GPU %d: miner not compiled for CUDA architecture %d.\n", ctx->device_id, gpuArch);
+			printf("SKIP! (arch not included)\n");
```

Comment:
please revert this

Reply:
Same as above, duplicate once the header is in